src/applications/personalization/profile360/reducers/index.js

```diff
@@ -94,11 +96,18 @@ function vaProfile(state = initialState, action) {
         state,
       );
 
+    case PAYMENT_INFORMATION_FETCH_STARTED:
     case PAYMENT_INFORMATION_SAVE_STARTED:
       return set('paymentInformationUiState.isSaving', true, state);
 
+    case PAYMENT_INFORMATION_FETCH_FAILED:
     case PAYMENT_INFORMATION_SAVE_FAILED: {
-      const newState = set('paymentInformationUiState.isSaving', false, state);
+      let newState = set(
```

Comment:
I'm a little concerned about conflating these two failure states, though I'm not too familiar with the UI. Is the new `set` call you added changing the UI when a save fails?

Reply:
Fetch failure reducer now separated.

Reply:
@jbalboni @erikphansen , there's a submission error though in master branch -- not sure if we need to fix via a separate bug ticket.

This error screenshot was taken running master branch:
<img width="1427" alt="master-pmt-info-update-submit-errors-190702a" src="https://user-images.githubusercontent.com/587583/60553085-97850880-9ce6-11e9-8539-bf59bf651c13.png">

Reply:
@tlei123 so it looks like: 1. we might not be handling 500s from the `PUT payment_information` endpoint, which is probably a problem. 2. is there any chance you were running [this branch](https://github.com/department-of-veterans-affairs/vets-api-mockdata/pull/86) of vets-api locally?

Reply:
Oh nevermind! That was for mocking a 500 on `GET` not `PUT`

Reply:
Error no longer comes up on-submission today, after pulling latest master.

Checked both master after pulling and my branch after merging master.